boards/arm/nucleo_g0b1re/arduino_r3_connector.dtsi

```diff
@@ -16,8 +16,8 @@
 			   <3 0 &gpiob 1 0>,	/* A3 */
 			   <4 0 &gpiob 11 0>,	/* A4 */
 			   <5 0 &gpiob 12 0>,	/* A5 */
-			   <6 0 &gpioa 3 0>,	/* D0 */
-			   <7 0 &gpioa 2 0>,	/* D1 */
+			   <6 0 &gpioc 5 0>,	/* D0 */
+			   <7 0 &gpioc 4 0>,	/* D1 */
```

Comment:
The nucleo_g071rb, uses the same board, could you eventually fix it there as well?